pod/perldiag.pod

```diff
@@ -3488,9 +3488,12 @@ handle.  This restriction may be eased in a future release.
 
 =item Lost precision when %s %f by 1
 
-(W imprecision) The value you attempted to increment or decrement by one
-is too large for the underlying floating point representation to store
-accurately, hence the target of C<++> or C<--> is unchanged.  Perl issues this
+(W imprecision) You attempted to increment or decrement a value by one,
+but the result is too large for the underlying floating point
+representation to store accurately. Hence, the target of C<++> or C<-->
+is increased or decreased by quite different value than one, such as
```

Comment:
This is poor English grammar.  I suggest changing 'quite' to 'a'

Reply:
Thank you for your advice.
I think it is better to say "significantly different from one", because no warnings are issued when amount of increase/decrease was slightly different from one (for example, `$a = 3.1; $a++;`).  So my revised proposal is:

```suggestion
is increased or decreased by an amount significantly different from one, such as
```